beta/src/components/Search.tsx

```diff
@@ -86,6 +90,13 @@ export const Search: React.FC<SearchProps> = ({
 
   useDocSearchKeyboardEvents({isOpen: isShowing, onOpen, onClose});
 
+  React.useEffect(() => {
+    if (typeof navigator !== 'undefined') {
+      setShowMacosShortcut(/(Mac|iPhone|iPod|iPad)/i.test(navigator.userAgent));
```

Comment:
Why are we testing for iPhone iPod? we do not show the shortcuts in those devices right ? 🤔 any specific reason ?

Reply:
When testing on my iPhone in landscape mode the shortcut key icons still display (happening on beta.reactjs.org and the preview deployment for this PR). 

If we don't test for iPhone/iPod, it would show the Ctrl key instead. Maybe a different breakpoint size is needed?

Reply:
> When testing on my iPhone in landscape mode the shortcut key icons still display (happening on beta.reactjs.org and the preview deployment for this PR).
> 
> If we don't test for iPhone/iPod, it would show the Ctrl key instead. Maybe a different breakpoint size is needed?

For landscape mode we should't show command key or crtl rather can we add orientation in the tailwind and handle for smaller iphone and ipod's landscape mode?  `landscape: {raw: '(orientation: landscape)'}`

It should work I guess.
let me know your thoughts 😄